app/components/clientsideonly/jitsibroadcaster.js

```diff
@@ -39,6 +39,9 @@ const Jitsibroadcaster = ({ room, disName, rtmpSrc, handleChat }) => {
     { speaker: "Z", hour: "50" },
   ];
 
+  //Set "showLogs" to true if you want to see logs for debugging
+  const showLogs = false;
```

Comment:
showLogs shouldn't be a constant in the components code!

Reply:
@Sing-Li I have changed the approach. The logs can now be toggled using a button in the dropdown.